packages/ckb-sdk-core/src/index.ts

```diff
@@ -116,6 +117,45 @@ class Core {
     return this.config.secp256k1Dep
   }
 
+  public loadDaoDep = async () => {
+    const genesisBlock = await this.rpc.getBlockByNumber('0x0')
+
+    /* eslint-disable */
```

Comment:
What about disabling specific violated rule?

Reply:
Updated in https://github.com/nervosnetwork/ckb-sdk-js/pull/382/commits/ea413cacae23af01289ee9086ff594f1af191740